oauth2_client/crypto.py

```diff
@@ -0,0 +1,43 @@
+"""
+Cryptographic utilities.
+"""
+from base64 import urlsafe_b64encode
```

Comment:
Yes I was looking but wasn't able to find a higher-level routine to do that job. This code is pretty much a copy paste of this example, only using SHA-256 instead of SHA. I am feeling 100% comfortable with that function as it only connects together a couple of blocks built by guys hopefully smarter than me.

https://www.dlitz.net/software/pycrypto/api/2.6/Crypto.Signature.PKCS1_v1_5-module.html

Reply:
@labstersteve  maybe you have heard about a ready util for this job?

Reply:
Does [PyJWT](https://pyjwt.readthedocs.io/en/latest/usage.html) work for you?

Reply:
Bad news:
PyJWT doesn't cut it. They don't provide interface to just sign the data, only to do encryption. Using their guts boils down to pretty much what I did.
Good news:
During the research I found a supposedly more renowned crypto library so switched to it (from pycryptodome to cryptography)